docs/user-guide/sources.md

```diff
@@ -0,0 +1,50 @@
+# Managing Sources
+
+Sources are repositories hosting prompt bundles.
+
+## Automatic Source Setup
+
+On first launch, Prompt Registry automatically adds the **Awesome Copilot** source (`github/awesome-copilot`). This gives you immediate access to community-curated collections without manual configuration.
+
+If you select a hub during first-run setup, all sources defined in that hub are also automatically synced.
+
+## Source Types
+
+| Type | Use Case | Status |
+|------|----------|--------|
+| `awesome-copilot` | Community collections (GitHub-hosted) | Active |
+| `local-awesome-copilot` | Local collection development/testing | Active |
+| `github` | GitHub repository releases | Active |
+| `local` | File system directories | Active |
+| `apm` | APM package repositories | Active |
+| `local-apm` | Local APM packages | Active |
+| `gitlab` | GitLab repository releases | ⚠️ Deprecated |
+| `http` | HTTP/HTTPS bundle URLs | ⚠️ Deprecated |
+
+> **Deprecation Notice:** The `gitlab` and `http` source types are deprecated and will be removed in a future release. Migrate to `github` or `awesome-copilot` sources.
+
+## Adding a Source
+
+`Ctrl+Shift+P` → "Prompt Registry: Add Source"
+
+## Managing Sources
+
+In Registry Explorer:
+- **Sync** — Right-click → Sync Source
+- **Edit** — Right-click → Edit Source
+- **Toggle** — Right-click → Toggle Enabled/Disabled
+- **Remove** — Right-click → Remove Source
+
```

Comment:
There is also "Open Repository"
.. and this link is available in all the context menus for all types of objects whenever it makes sense